modules/travel_pay/app/services/travel_pay/client.rb

```diff
@@ -182,7 +182,7 @@ def connection(server_url:)
 
       Faraday.new(url: server_url) do |conn|
         conn.use :breakers
-        conn.response :raise_error, error_prefix: service_name, include_request: true
+        conn.response :raise_custom_va_error, error_prefix: service_name, include_request: true
```

Comment:
@kjduensing I'm including you (as a data point, not to cast blame) as an example of why this poorly named class is confusing. Can I assume that when I next update the default value of `include_request` to false, you'll want me to add `raise_error,  include_request: true` to the new chain? I'll keep you in the loop on that PR.

Note: To be clear, the current inclusion of `include_request: true`, isn't actually doing anything, as it's a default option of the Faraday class, not our class